src/opnsense/mvc/app/models/OPNsense/Routes/Route.xml

```diff
@@ -0,0 +1,24 @@
+<model>
+    <mount>//staticroutes</mount>
+    <version>0.0.1</version>
+    <items>
+        <route type="ArrayField">
+          <network type="NetworkField">
+            <NetMaskRequired>Y</NetMaskRequired>
+            <Required>Y</Required>
+          </network>
+          <gateway type="JsonKeyValueStoreField">
+            <Required>Y</Required>
+            <ConfigdPopulateAct>interface gateways list</ConfigdPopulateAct>
+            <SourceFile>/tmp/gateway_list.json</SourceFile>
```

Comment:
@fabianfrz you probably want to set ConfigdPopulateTTL to a low value here, to avoid caching, see https://github.com/opnsense/core/blob/master/src/opnsense/mvc/app/models/OPNsense/Base/FieldTypes/JsonKeyValueStoreField.php#L166-L171